File: Selenium/Project/WebsiteSecondHeading.java
package Project;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class WebsiteSecondHeading {
        private WebDriver driver;
        private String baseUrl = "https://alchemy.hguy.co/jobs/";

        @BeforeClass
        public void setUp() {
            System.setProperty("webdriver.gecko.driver",  "C://Users//001ZXG744//IdeaProjects//geckodriver.exe");
            driver = new FirefoxDriver();
            driver.get(baseUrl);
        }

        @Test
        public void testSecondHeading() {
            // Get the second heading on the page
            WebElement secondHeadingElement = driver.findElement(By.xpath("//h3[@class='entry-title']/a"));
            String actualSecondHeading = secondHeadingElement.getText();

            // Verify the second heading matches the expected text
            String expectedSecondHeading = "Quia quis non";
            Assert.assertEquals(actualSecondHeading, expectedSecondHeading, "Second heading does not match the expected text.");
        }

        @AfterClass
        public void tearDown() {
            driver.quit();
        }
    }

File: Selenium/Project/WebsiteSecondHeading.py
import unittest
from selenium import webdriver
from selenium.webdriver.common.by import By

class WebsiteSecondHeading(unittest.TestCase):
    def setUp(self):
        self.driver = webdriver.Firefox(executable_path="C:/Users/001ZXG744/IdeaProjects/geckodriver.exe")
        self.base_url = "https://alchemy.hguy.co/jobs/"

    def test_second_heading(self):
        # Navigate to the base URL
        self.driver.get(self.base_url)

        # Get the second heading on the page
        second_heading_element = self.driver.find_element(By.XPATH, "//h3[@class='entry-title']/a")
        actual_second_heading = second_heading_element.text

        # Verify the second heading matches the expected text
        expected_second_heading = "Quia quis non"
        self.assertEqual(actual_second_heading, expected_second_heading, "Second heading does not match the expected text.")

    def tearDown(self):
        self.driver.quit()

if __name__ == "__main__":
    unittest.main()
